mne_bids/tests/test_utils.py

```diff
@@ -9,18 +9,41 @@
 from datetime import datetime
 
 from scipy.io import savemat
+from numpy.random import random
 import mne
 from mne.datasets import testing
 from mne.utils import _TempDir
 
 from mne_bids.utils import (make_bids_folders, make_bids_basename,
                             _check_types, print_dir_tree, _age_on_date,
                             _get_brainvision_paths, copyfile_brainvision,
-                            copyfile_eeglab, _infer_eeg_placement_scheme)
+                            copyfile_eeglab, _infer_eeg_placement_scheme,
+                            _handle_kind)
 
 base_path = op.join(op.dirname(mne.__file__), 'io')
 
 
+def test_handle_kind():
```

Comment:
1. it increases coverage
2. it makes sure that MEG, iEEG and EEG kinds are discovered properly (if single meg channel=MEG, is single ecog channel=iEEG, if eeg and no meg or ecog=EEG). Basically this test covers the changes in order I made to `_handle_kind`

I think it's good to have because the `kind` is an important variable in our code